app/helpers/application_helper.rb

```diff
@@ -239,20 +239,6 @@ def avatar_image_tag(user, html_options = {})
     end
   end
 
-  def readonly_field(object, property, options = {})
```

Comment:
For the record: this method was created for the old settings page and isn't used anymore.